app-5-rdf-serializer/src/test/java/org/geovistory/toolbox/streams/rdf/processors/project/ProjectOwlClassTest.java

```diff
@@ -0,0 +1,228 @@
+        var outRecords = outputTopic.readKeyValuesToMap();
+        assertThat(outRecords).hasSize(2);
+    }
+
+    /**
+     * The goal of this test is to check if the operation value (insert or delete) is correctly set in the output topic
+     */
+    @Test
+    void testOperationValue() {
+
+        var projectId = 1;
+        var classId = 123212;
+        var label = "test";
+        var lang = "en";
+
+        var k = ProjectClassLabelKey.newBuilder().setProjectId(projectId).setClassId(classId).build();
+        var v = ProjectClassLabelValue.newBuilder()
+                .setProjectId(projectId)
+                .setClassId(classId)
+                .setLabel(label)
+                .setLanguageIso(lang)
+                .setLanguageId(I.EN.get())
+                .setDeleted$1(false)
+                .build();
+        projectClassLabel.pipeInput(k, v);
+
+        var outRecords = outputTopic.readKeyValuesToMap();
+
+        var expectedKey = ProjectRdfKey.newBuilder()
+                .setProjectId(projectId)
+                .setTurtle("<https://ontome.net/ontology/c123212> a <http://www.w3.org/2002/07/owl#Class> .")
+                .build();
+
+        var record = outRecords.get(expectedKey);
+        assertThat(record.getOperation()).isEqualTo(Operation.insert);
+
+        var expectedKey2 = ProjectRdfKey.newBuilder()
+                .setProjectId(projectId)
+                .setTurtle("<https://ontome.net/ontology/c123212> <http://www.w3.org/2000/01/rdf-schema#label> \"test\"@en .")
+                .build();
+
+        var record2 = outRecords.get(expectedKey2);
+        assertThat(record2.getOperation()).isEqualTo(Operation.insert);
+
+        var v2 = ProjectClassLabelValue.newBuilder()
+                .setProjectId(projectId)
+                .setClassId(classId)
+                .setLabel(label)
+                .setLanguageIso(lang)
+                .setLanguageId(I.EN.get())
+                .setDeleted$1(true)
+                .build();
+        projectClassLabel.pipeInput(k, v2);
+        outRecords = outputTopic.readKeyValuesToMap();
+
+        expectedKey = ProjectRdfKey.newBuilder()
+                .setProjectId(projectId)
+                .setTurtle("<https://ontome.net/ontology/c123212> a <http://www.w3.org/2002/07/owl#Class> .")
+                .build();
+
+        record = outRecords.get(expectedKey);
+        assertThat(record.getOperation()).isEqualTo(Operation.delete);
+
+        expectedKey = ProjectRdfKey.newBuilder()
+                .setProjectId(projectId)
+                .setTurtle("<https://ontome.net/ontology/c123212> <http://www.w3.org/2000/01/rdf-schema#label> \"test\"@en .")
+                .build();
+
+        record = outRecords.get(expectedKey);
+        assertThat(record.getOperation()).isEqualTo(Operation.delete);
+    }
+
+    /**
+     * The goal of this test is to check if the key/value pairs generated in the processor match the expected list of key/value pairs
```

Comment:
I suggest to add in the comment:

The expectation is that special characters " and \ are properly escaped.

Reply:
Done